tools/ci_connector_ops/ci_connector_ops/pipelines/tests/__init__.py

```diff
@@ -28,6 +29,18 @@
 }
 
 
+async def run_metadata_validation(context: ConnectorContext) -> List[StepResult]:
```

Comment:
I like this! Should we remove the metadata validation CI step then?

Reply:
I like to keep the step class as it can allow us to compose pipelines. I declared this function for consistency with how we originally call static analysis checks.